change/@fluentui-react-native-button-2020-05-13-14-36-09-sammy-finalAppium.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
A change like this should probably be a patch version update. Because we're pre 1.0.0, I'm not sure how beachball resolves the version numbers. There's probably a way to update your change jsons via some beachball command.

Reply:
I've looked at the Beachball docs and can't find a command that allows you to update your change jsons. What would you suggest my next steps forward are?

Reply:
Would manually changing this file have any effect?

Reply:
manually changing the files is probably fine - I don't think there are additional fields under different types

Reply:
Changed in newest commit